### ISSUE TITLE
Star Ocean: Second Evolution: Text rendering problems

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#17354]
Reported by Pandavenom on Discord.

![image](https://user-images.githubusercontent.com/130929/235300931-93005edd-06d4-40a8-966f-4186b5aa2aee.png)
![image](https://user-images.githubusercontent.com/130929/235300934-59af8b2a-38fa-4856-a827-9e394bb29082.png)

It looks correct in the software renderer though:

![image](https://user-images.githubusercontent.com/130929/235301640-a60e7af5-0ac8-4206-af76-925a3ddff15d.png)

intel cpu (12600K), and an rtx3060 card. Vulkan, D3D and Ogl are the same.

"when I open the menu ingame, it works fine first, but after I enter a submenu the text won't work anymore"

This means it's likely some framebuffer confusion or framebuffer offsets..

A [GE frame capture](https://github.com/hrydgard/ppsspp/wiki/How-to-create-a-frame-dump) would be very useful.

**Comment by Pandavenom** [comment]
I hope this will help! :)

[ULUS10375_0001.zip](https://github.com/hrydgard/ppsspp/files/11359126/ULUS10375_0001.zip)

**Comment by unknownbrackets** [comment]
I can't reproduce this.  As noted on Discord:

I opened the game, went into the menu, and selected status on CLAUDE.  Everything looked fine.

I also tried changing his name, no effect.  I went back and tried other submenus: Save, Strategy, Learn Skills, Equipment, Special Arts... but everything was fine after going to those.

I did this with the regular US version, it sounds like this is happening with a patched version of the game (undub.)  Not sure if it only happens there or if there are more settings involved.

A save state would help in case it only happens in certain areas.

-[Unknown]
